buildbot/master/files/config/environments.py

```diff
@@ -152,4 +153,5 @@ def without(self, to_unset):
     'AWS_ACCESS_KEY_ID': S3_UPLOAD_ACCESS_KEY_ID,
     'AWS_SECRET_ACCESS_KEY': S3_UPLOAD_SECRET_ACCESS_KEY,
     'GITHUB_HOMEBREW_TOKEN': GITHUB_HOMEBREW_TOKEN,
+    'WPT_SYNC_TOKEN': WPT_SYNC_PR_CREATION_TOKEN,
```

Comment:
Why are we putting this in the `upload_nightly` environment? It sounds like this is being run as a separate step, so we should use a separate environment (e.g. I find it unlikely this needs AWS access), and use the environment combining feature with `+` if necessary.

Reply:
Fixed.